README.md

```diff
@@ -38,7 +38,7 @@ SlimFAQSDK.shared.set(clientID: "slimwiki")
 // present SlimFAQ screen
 do {
     try SlimFAQSDK.shared.present(from: self, animated: true, completion: nil)
-} catch {
+} catch let error {
```

Comment:
@aunnnn actually there's no typo. If `catch` clause was written without any pattern it will automatically bind an error with type `Any` as a local var named **error**. More detailed explanation is here - [https://docs.swift.org/swift-book/LanguageGuide/ErrorHandling](https://docs.swift.org/swift-book/LanguageGuide/ErrorHandling.html#ID541)
Anyway I will add your change since other users also might be confused. Thanks for catching this!

Reply:
@freemansion Wow never knew that before. Good to know 👍